client/asset/eth/eth.go

```diff
@@ -126,7 +123,12 @@ func (d *Driver) Info() *asset.WalletInfo {
 
 // Exists checks the existence of the wallet.
 func (d *Driver) Exists(walletType, dataDir string, settings map[string]string, net dex.Network) (bool, error) {
-	return false, errors.New("unimplemented")
+	if walletType != walletTypeGeth {
+		return false, fmt.Errorf("wallet type %q unrecognized", walletType)
+	}
+	keyStoreDir := filepath.Join(getWalletDir(dataDir, net), "keystore")
+	ks := keystore.NewKeyStore(keyStoreDir, keystore.LightScryptN, keystore.LightScryptP)
```

Comment:
There are three different magic values, including the new "keystore".  Can we make these package-level?